### PR TITLE
update roadmap

### PR DESCRIPTION
**Pull request by deepthi**
- Update roadmap with known scope for the next 6-9 months.
- Remove Postgres

### CODE REVIEW

File: content/en/docs/resources/roadmap.md [hunk 1]
```diff
@@ -10,25 +10,37 @@ As an open source project, Vitess is developed by a community of contributors. M
...
+  - Declarative schema
+- VSchema improvements
+  - Vtgates auto-detect schema changes
+- Vitess-native unplanned failovers
+- Pluggable durability policies
 - Rewrite of vtctld UI including visualization of VReplication
 - VReplication throttling
-- Binlog server
 - Topology Service: Reduce dependencies on the topology service. i.e. Vitess should be operable normally even if topology service is down for several hours. Topology service should be used only for passive discovery.
-- Support for PostgreSQL: Vitess should be able to support PostgreSQL for both storing data, and speaking the protocol in VTGate.
```
Comment (guidoiaquinti) [inline content/en/docs/resources/roadmap.md:34]: Is this going to be dropped completely?

**Comment by guidoiaquinti** [comment]
Should we add a "Long term" section for items not planned for the short-mid term?

**Comment by deepthi** [comment]
> Should we add a "Long term" section for items not planned for the short-mid term?

We _could_ but it seems more aspirational than anything else.
For instance we have had Postgres in the roadmap for almost 2 years but made no progress.
The roadmap should be useful for people adopting vitess, and I feel that a vague long-term roadmap isn't very helpful.

File: content/en/docs/resources/roadmap.md (hunk 1 above)
Comment (deepthi) [reply content/en/docs/resources/roadmap.md:34]: https://github.com/vitessio/vitess/issues/7084#issuecomment-848289310

File: content/en/docs/resources/roadmap.md [hunk 2]
```diff
@@ -10,25 +10,37 @@ As an open source project, Vitess is developed by a community of contributors. M
...
+## Medium Term (3-9 months)
 
-- Vttablet to manage more than one MySQL schema
+- MySQL compatibility
+  - More frameworks
+- Query Planning improvements
+  - Performance
+  - More supported queries
+- Schema changes
+  - Declarative schema
```
Comment (shlomi-noach) [inline content/en/docs/resources/roadmap.md]: We can remove this as [declarative migrations](https://vitess.io/docs/user-guides/schema-changes/declarative-migrations/) are already published; I'll be waiting for community feedback/input and have no immediate further plans for declarative schemas.

File: content/en/docs/resources/roadmap.md [hunk 3]
```diff
@@ -10,25 +10,37 @@ As an open source project, Vitess is developed by a community of contributors. M
...
-## Medium Term
+## Medium Term (3-9 months)
 
-- Vttablet to manage more than one MySQL schema
+- MySQL compatibility
+  - More frameworks
+- Query Planning improvements
+  - Performance
+  - More supported queries
+- Schema changes
```
Comment (shlomi-noach) [inline content/en/docs/resources/roadmap.md:38]: I'd add a `- Usability` bullet, because we are starting to address community (and internal) feedback

**Review by shlomi-noach — COMMENTED** [review]
couple suggestions for change

**Review by rohit-nayak-ps — APPROVED** [review]
lgtm

**Pull request merged**